elm/day1/src/Calibration.elm

```diff
@@ -1,5 +1,26 @@
 module Calibration exposing (..)
 
+import Iter
+import Iter.Extra
+import List.More
+
 
 calculate =
     List.foldl (+) 0
+
+
+calibrate changes =
+    changes
+        |> Iter.cycle
+        |> Iter.accumulate (+) 0
+        |> Iter.accumulate (::) [ 0 ]
+        |> Iter.map (Debug.log "step")
+        |> Iter.map List.More.duplicates
+        |> Iter.Extra.dropWhile List.isEmpty
```

Comment:
Basic idea here is:
1. iterate changes input forever
2. build a running sum to generate frequencies at each step
3. build a running cons to generate a growing set of frequency changes at each step
4. check for duplicates at each step
5. stop at the first non-empty duplicate list
6. unwind to find the first duplicate frequency